language/stdlib/modules/Genesis.move

```diff
@@ -28,9 +28,10 @@ module Genesis {
 
     fun initialize(
         lr_account: &signer,
+        lr_auth_key: vector<u8>,
```

Comment:
nit: we try to have all the signers as the first argument, so could you move this to be right next to the `tc_auth_key`?